src/nspawn/nspawn.c

```diff
@@ -3289,6 +3291,12 @@ static int outer_child(
 
         log_debug("Outer child is initializing.");
 
+        r = load_os_release_pairs_with_prefix("/",
+                                              arg_start_mode == START_BOOT ? "container_host_" : "service_host_",
+                                              &os_release_pairs);
```

Comment:
as mentioned. Let's only import the four _ID fields, nothing else. Who knows what crazy stuff people put in `/etc/os-release` after all. Let's only use the minimal info, i.e. the computer readable stuff that we know well, and is truly useful.

And I think I changed my mind on the `service_` prefix: i don't think we should use env vars for propagating this data to "services" at all, for the reasons people pointed out in the discussion.

Thus, let's only set `$container_host_id=` (which is set to whatever ID= is set to), `$container_host_version_id=` (from VERSION_ID=`), `$container_host_build_id` (from BUILD_ID), `$container_host_variant_id=` from (VARIANT_ID), and nothing else, and do so only in nspawn.

For passing such data to services, let's adopt the /run/host thing that flatpak/bubblewrap use.